src/node_dotenv.cc

```diff
@@ -8,6 +9,17 @@ namespace node {
 using v8::NewStringType;
 using v8::String;
 
+/**
+ * The inspiration for this implementation comes from the original dotenv code,
```

Comment:
Does this mean that the parser is exactly compatible with `dotenv`? https://github.com/nodejs/node/issues/49148#issuecomment-1792881224 sounds like `dotenv` supports multi-line values, and if it does, this implementation should probably be compatible.

Reply:
@tniessen The goal of this PR is to support multi-line values same as `dotenv`